### PR TITLE
Create natur-und-landschaft.csl

### PR DESCRIPTION
**Pull request by damnation333**
via https://forums.zotero.org/discussion/56268/style-request-natur-und-landschaft-german-journal#latest

Inconsistent style in the two examples that were provided (even numeric vs author-date). I just stuck to the guidelines now.

### CODE REVIEW

**Comment by damnation333** [comment]
I don't get why Travis is grumpy?

File: natur-und-landschaft.csl [hunk 1]
```diff
@@ -0,0 +1,207 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="de-DE">
+  <info>
+    <title>Nature and Lancscape (Natur und Landschaft, German)</title>
```
Comment (rmzelle) [inline natur-und-landschaft.csl]: Typo: Lancscape

File: natur-und-landschaft.csl [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+    <names variable="author">
+      <name form="short" delimiter-precedes-last="never" initialize-with=". "/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <text macro="anon"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="title"/>
```
Comment (rmzelle) [inline natur-und-landschaft.csl]: Travis doesn't like empty macros. (also not valid CSL)

**Comment by rmzelle** [comment]
> must define all macros that are called by `<text/>` and `<key/>` elements
     Failure/Error: `expect(style.macros).to have_key(node[:macro])`
       `expected #has_key?("title")`

You don't need help with this one, right?

**Comment by rmzelle** [comment]
Those double parentheses around URLs are ... special.

Was there any particular reason you included the English translation of the title? For journals that publish in a single language we typically don't include translations.

**Comment by rmzelle** [comment]
And thanks!

**Pull request merged**